03-reverseArray.js

```diff
@@ -4,4 +4,16 @@
  * Task Complexity: 1 of 5
  * @param {Array} array - Array of any elements
  * @returns {Array}
- */
\ No newline at end of file
+ */
+function reverseArr(arr1) {
```

Comment:
Usually we don't add numbers, so it's better to leave just `arr`

```suggestion
function reverseArr(arr) {
```